engine/src/flutter/lib/ui/painting.dart

```diff
@@ -2460,10 +2464,6 @@ Future<Codec> instantiateImageCodec(
 /// The data can be for either static or animated images. The following image
 /// formats are supported: {@macro dart.ui.imageFormats}
 ///
-/// The [buffer] will be disposed by this method once the codec has been created,
```

Comment:
If `instantiateImageCodecWithSize` is restored, then this comment should not be removed.